RELEASE/scripts/autoscend/auto_combat.ash

```diff
@@ -590,13 +590,20 @@ string auto_combatHandler(int round, string opp, string text)
 
 	if(enemy.to_string() == "skeleton astronaut")
 	{
+		if(my_daycount() == 1 && item_amount($item[Exploding cigar]) > 0){
+			return "item " + $item[Exploding cigar];
+		}
 		int dmg = 0;
 		foreach el in $elements[hot, cold, sleaze, spooky, stench]
 		{
 			dmg += min(10, numeric_modifier(el.to_string() + " Damage"));
```

Comment:
theres also a Prismatic Damage modifier, but i dont know details for it (might not be necessary)

Reply:
I'm not sure of the details (does it give the minimum of the 5 elemental damages? does it give the sum?) but I think the existing logic's intent is pretty clear. Open to revisiting this later if this logic gets more complicated.